Golden_Path.postman_collection.json

```diff
@@ -9232,6 +9234,10 @@
 													"              pm.test(\"fspiop-destination is payerfsp\", function () {",
 													"                pm.expect(headers['fspiop-destination']).to.eql('payerfsp');",
 													"              });",
+													"              ",
+													"              pm.test(\"fspiop-uri is empty\", function () {",
+													"                pm.expect(headers['fspiop-uri']).to.eql(`/transfers/${transferId}`);",
```

Comment:
Update the test name please - its not empty now

Reply:
Updated the comments and rerun of the collection - same results.